glyphs2fontir/src/source.rs

```diff
@@ -480,9 +480,9 @@ mod tests {
 
         assert_eq!(
             HashSet::from([
-                &BTreeMap::from([("wght".to_string(), NormalizedCoord::new(-1.0))]),
-                &BTreeMap::from([("wght".to_string(), NormalizedCoord::new(0.0))]),
-                &BTreeMap::from([("wght".to_string(), NormalizedCoord::new(1.0))]),
+                &NormalizedLocation::on_axis("wght", NormalizedCoord::new(-1.0)),
```

Comment:
If the string was a tag, this type could derive `Copy`, and you wouldn't need to be borrowing here.

Reply:
Oh dear, this test is horribly misleading, it should say things like Weight not things like wght

Reply:
https://github.com/googlefonts/fontmake-rs/issues/70 for types to make this a little harder to bungle

Reply:
In this PR I have updated to make use of name consistent